cmd/store/query/query.go

```diff
@@ -1,3 +1,5 @@
+//+build ignore
```

Comment:
Are you ignoring these just to make it build?

Do we have issues to fix commands?

Reply:
We've apparently fixed it on InfluxDB 2.0 and it's an internal command that uses a now removed interface according to @stuartcarnie so he said it was fine if I just ignore the files for now. Since it's not part of the release, I figured it would be fine.